tests/unittests/Foundation/NSProgressTests.mm

```diff
@@ -133,7 +135,9 @@ - (void)addParentTo:(NSProgress*)child {
                      [userInfo setCompletedUnitCount:1];
                  }
         andExpectChangeCallbacks:nil];
-    EXPECT_EQ(2, kvoListener.hits);
+
+    // Depending on OSX version, the implementation differs slightly, and changes the number of hits
+    EXPECT_TRUE((kvoListener.hits == 1) || (kvoListener.hits == 2));
```

Comment:
i'd prefer to err on too-specific here if that's okay

---
In reply to: [95452816](https://github.com/Microsoft/WinObjC/pull/1663#discussion_r95452816) [](ancestors = 95452816)